apps/renterd/contexts/config/useRedundancyMultiplier.tsx

```diff
@@ -6,8 +6,8 @@ export function useRedundancyMultiplier({
   minShards,
   totalShards,
 }: {
-  minShards: BigNumber
-  totalShards: BigNumber
+  minShards?: BigNumber
```

Comment:
I understand most of the changes but this one looks a bit off. Why would we want to allow optional parameters here?

Reply:
This method and many others have to have optional params to handle when the user clears the field, or if the data is still loading from the server, both of which would result in undefined values - it may be more accurate to make this minShards: BigNumber|undefined meaning the caller has to pass/provide the param, but it can be undefined.